app/src/main/res/values/strings.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2" xmlns:tools="http://schemas.android.com/tools">
+<resources xmlns:tools="http://schemas.android.com/tools" xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
     <string name="action_search">Search</string>
     <string name="action_add">Add</string>
     <plurals name="selectedCardCount">
         <item quantity="one"><xliff:g>%d</xliff:g> card selected</item>
         <item quantity="other"><xliff:g>%d</xliff:g> cards selected</item>
     </plurals>
     <string name="noGiftCards">Click the + plus button to add a card, or import some from the ⋮ menu first.</string>
+    <string name="noGiftCardsGroup">Looks like you have not added any loyalty cards yet. They can be added to this group from here once you have some ^w^</string>
```

Comment:
Smileys are awful for screen readers, so let's not use any.

How about:

"You don't have any loyalty cards yet. Once you've added some you can add them to the group here."